geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Cu   Cl     2.0614(7)
+Cu...N{H3}  1.9182(13)
+N    H      1.0186
+H    N...Cu 111.40(6)
+
+# Bond lengths: rs     (angstroms)
+# Bond angles:  thetas (degrees)
+
+Cu   Cl     2.058(4)
+Cu...N{H3}  1.918(4)
+H    N...Cu 111.5(1)
+
+NAME = ClHZn
+# Point Group: Coov
+# Shape: Linear
+# Bond lengths: r0     (angstroms)
+# Source Paper: Pulliam RL, Sun M, Flory MA, Ziurys LM (2009) The sub-millimeter and Fourier transform microwave
+spectrum of HZnCl (X1 Σ+). J Mol Spectrosc 257(2):128-132
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H  Zn 1.519
+Zn Cl 2.083
+
+# Bond lengths: rs     (angstroms)
+
+H  Zn 1.519
+Zn Cl 2.082
+
+# Bond lengths: rm^(2)     (angstroms)
+
+H  Zn 1.5050(17)
+Zn Cl 2.08293(21)
+
+NAME = ClH3Si
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Cazzoli G, Puzzarini C, Gauss J (2012) Rotational spectrum of silyl chloride: hyperfine structure and equilibrium
+geometry. Mol Phys 110(19-20):2359-2369
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Si Cl   2.04584(6)
+Si H    1.46878(16)
+Cl Si H 108.427(13)
+
+NAME = Cl2OS
+# Point Group: Cs
+# Shape: Trigonal pyramidal
+# Bond lengths:     r0     (angstroms)
+# Bond angles:      theta0 (degrees)
+# Dihedral angles:  tau0   (degrees)
+# Source Paper: Martin-Drumel MA, Roucou A, Brown GG, Thorwirth S, Pirali O, Mouret G, Hindle F, McCarthy MC, Cuisset
+A (2016) High resolution spectroscopy of six SOCl2 isotopologues from the microwave to the far-infrared. J
+Chem Phys 144(8):084305/1-084305/7
+[http://dx.doi.org/10.1063/1.4942024]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+S = O           1.434(9)
+S   Cl          2.072(3)
+Cl  S  = O      108.00(6)
+Cl  S    Cl     97.08
+Cl  S  = O...Cl 104.0(3) 
+
+# Bond lengths:     re^se     (angstroms)
+# Bond angles:      thetae^se (degrees)
+# Dihedral angles:  taue^se   (degrees)
+
+S = O           1.4340(7)
+S   Cl          2.0700(3)
+Cl  S  = O      107.703(6)
+Cl  S    Cl     96.929
+Cl  S  = O...Cl 103.58(3)
+
+NAME = Cl4Te
+# Point Group: C2v
+# Shape: Seesaw
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetag (degrees)
+# Source Paper: Shlykov SA, Giricheva NI, Titov AV, Szwak M, Lentz D, Girichev GV (2010) The structures of
+tellurium(IV) halides in the gas phase and as solvated molecules. Dalton Trans 39 (13):3245-3255
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Te     Cl(ax)    2.428(4)
+Te     Cl(eq)    2.289(3)
+Cl(ax) Te Cl(ax) 176.7(10)
+Cl(eq) Te Cl(eq) 102.5(7)
+
+NAME = CuFH2
+# Point Group: C2v
+# Shape: Linear --- Linear
+# Bond lengths: rs     (angstroms)
+# Source Paper: Frohman DJ, Grubbs GS, Yu Z, Novick SE (2013) Probing the chemical nature of dihydrogen complexation to
+transition metals, a gas phase case study: H2-CuF. Inorg Chem 52(2):816-822
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Cu F   1.7409(1)
+H2 CuF 1.5246(5)
```

Comment:
"H2 CuF" seems to imply that we have the distance between the centre of masses of H2 and CuF. 

Instead, "H2 Cu" would indicate that we have the distance between H2 ad Cu.

Reply:
I don't think you are right here.
![image](https://github.com/HPQC-LABS/goDatabase/assets/123907626/7b6883b5-64c0-46c2-b639-f65695395710)
It does say that Rcm is the "Distance between the centers of mass of the monomer subunits. "

Reply:
@sichaox Thanks for pointing that out. Now how will the computer know the orientation of the two monomer units with respect to each other? Does the point group alone provide enough information to know the orientation? Can a computer construct the z-matrix with the information given?

Reply:
It' C2v but then there are still 2 possibilities so the computer won't be able to know.

Reply:
It' C2v but then there are still 2 possibilities so the computer won't be able to know.